pom.xml

```diff
@@ -484,6 +484,16 @@
         <version>${guava.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>com.github.ben-manes.caffeine</groupId>
+        <artifactId>caffeine</artifactId>
+        <version>2.9.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.github.ben-manes.caffeine</groupId>
+        <artifactId>guava</artifactId>
+        <version>2.9.0</version>
+      </dependency>
```

Comment:
Any particular reason to use 2.x when 3.x is out ?

Reply:
because Java 11 or above is required for 3.x and it is no longer compatible with Java 8